influxdb-spring-3.0.0.jar has no class com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration

<img width="696" alt="Screen Shot 2021-07-15 at 4 06 09 PM" src="https://user-images.githubusercontent.com/10142314/125752612-e18ee14d-eea9-4e4b-a304-c80fe29fcf5e.png">

Then I guess `spring.factories` in influxdb-spring should not contain `com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration`, or we'll get this:

```
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: java.lang.IllegalStateException: Unable to read meta-data for class com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration
Caused by: java.io.FileNotFoundException: class path resource [com/influxdb/spring/metrics/Influx2MetricsExportAutoConfiguration.class] cannot be opened because it does not exist
```

--- a/spring/src/main/resources/META-INF/spring.factories
+++ b/spring/src/main/resources/META-INF/spring.factories
@@ -1,4 +1,3 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
 com.influxdb.spring.health.InfluxDB2HealthIndicatorAutoConfiguration,\
-com.influxdb.spring.influx.InfluxDB2AutoConfiguration,\
-com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration
+com.influxdb.spring.influx.InfluxDB2AutoConfiguration

